test/spec/features/bendpoints/BendpointsSpec.js

```diff
@@ -128,6 +128,31 @@ describe('features/bendpoints', function() {
     }));
 
 
+    it('should NOT activate on AUXILARY mouse button', inject(
```

Comment:
minor typo : "AUXILIARY"

Reply:
Renamed everywhere. Nice catch. Nearly made it into protected API :stuck_out_tongue_winking_eye:

Reply:
AXILARIOUS 🤣